### ISSUE TITLE
New Footer row for logos and social actions

### ISSUE THREAD

**Issue by tdg** [swapmyvote/swapmyvote#99]
Facebook 'like' and 'share'

Step 1: Include the JavaScript SDK on your page once, ideally right after the opening body tag.

```
<div id="fb-root"></div>
<script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_GB/sdk.js#xfbml=1&version=v5.0&appId=377810399053422"></script>```

Step 3: Place this code wherever you want the plugin to appear on your page.

```
<div class="fb-like" data-href="https://www.facebook.com/swapmyvote/" data-width="" data-layout="button_count" data-action="like" data-size="small" data-show-faces="false" data-share="true"></div>```

**Comment by tdg** [comment]
Twitter follow 

```
<a href="https://twitter.com/TwitterDev?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-screen-name="false" data-dnt="true" data-show-count="false">Follow @TwitterDev</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>```

**Comment by tdg** [comment]
Forward Democracy logo and link
![F](https://user-images.githubusercontent.com/95868/69368438-124cc100-0c92-11ea-9a34-37d0a8da673d.png)
![FORWAR EMOCRACY](https://user-images.githubusercontent.com/95868/69368440-12e55780-0c92-11ea-8c2a-cdd84c5403b1.png)
To link to `forwarddemocracy.com`

**Comment by tdg** [comment]
Forward Democracy 
Studio Microclimate
Great Northern Road 
Cambridge CB1 2FX 
Company number 11707599

**Comment by fkorning** [comment]
To recap, you want a floating footer bar that has buttons for social media:
 [FB like] [FB share] [Tweet this] [Fwd Dem] ?

**Comment by tdg** [comment]
Not sure about floating, but yes, a footer bar where we can put logos and those social actions (what is fwd dem?)

It should also have room for a logo in the centre and some text (impressum)

**Comment by fkorning** [comment]
[Fwd Dem] -> [Forward Democracy]  (or brand of choice - "or powered by foo...")

    On Monday, 2 December 2019, 13:57:30 GMT, Tom de Grunwald <notifications@github.com> wrote:  
 
 
Not sure about floating, but yes, a footer bar where we can put logos and those social actions (what is fwd dem?)

It should also have room for a logo in the centre and some text (impressum)

**Comment by fkorning** [comment]
how urgent is this?  I'd love to take it, but it will take a while...

**Comment by fkorning** [comment]
something like this?   do you still want the url links ?
![image](https://user-images.githubusercontent.com/8026632/70061974-1a372a00-15dd-11ea-95aa-e07525ebc404.png)

**Comment by tdg** [comment]
Yes, good start! I think the log should be central and [bigger](https://www.designershumor.com/wp-content/uploads/2019/07/66635375_151699899315232_7184747540541087857_n-768x550.jpg) and I think 'Powered by' is a good idea. 
The social actions could be less prominent - perhaps in a more subtle grey scale colour?
It's important that these never detract from the central call to action of the site

**Comment by tdg** [comment]
Social actions should be like and share on FB, follow and share on Twitter, email, whatsapp, telegram, linkedin, 'more'
I like this look:
![Screen Shot 2019-12-03 at 16 12 43](https://user-images.githubusercontent.com/95868/70068462-f2998f00-15e7-11ea-8db2-a104ed8b4523.png)

**Comment by fkorning** [comment]
![image](https://user-images.githubusercontent.com/8026632/70086101-90518600-1609-11ea-9dd6-152f75d90050.png)

**Comment by fkorning** [comment]
I'm going to save this in a branch and hand it off.  I'm not a webdev and I don't know enough of mvc, ruby, rails, and facebook/twitter apis to get this working.  I can do the uber basic button rendering, but after that I would get bogged down and slow this down...  I'm adding icons in assets for friendly sites should one want to link to them (aka powered by Electoral-Calculus, or Democracy-Club).   So passing it on to more competent hands..

**Comment by fkorning** [comment]
this is the state of it now... the buttons are just urls, so I haben't done the fancy javascript to link to fb.  they seem to want to force one to use their own buttons (fb-like) and I don't know the necessary app params to get this done (nor on twitter, etc). 

![image](https://user-images.githubusercontent.com/8026632/70393394-28bc8180-19e1-11ea-93f1-84a328ebc6cb.png)

**Comment by fkorning** [comment]
@tdg @aspiers : I scrounged some time and managed to get facebook, twitter, tumblr, and email working. ready to integrate

![image](https://user-images.githubusercontent.com/8026632/70605281-23825100-1bf2-11ea-82fe-a1a631e9dd1a.png)

**Comment by fkorning** [comment]
hang on.. merge broke things...

**Comment by fkorning** [comment]
fixed. reorderd buttons so color progresses from light to dark.

![image](https://user-images.githubusercontent.com/8026632/70631822-acff4680-1c25-11ea-98b9-d80fbf5c991a.png)
